rocky/crisis_room/views.py

```diff
@@ -22,21 +22,22 @@
 from tools.models import Organization, OrganizationMember
 
 from crisis_room.forms import AddDashboardForm
-from crisis_room.management.commands.dashboards import FINDINGS_DASHBOARD_NAME, get_or_create_dashboard
-from crisis_room.models import Dashboard, DashboardData
+from crisis_room.models import Dashboard, DashboardItem
 from octopoes.config.settings import DEFAULT_SCAN_LEVEL_FILTER, DEFAULT_SCAN_PROFILE_TYPE_FILTER
 from octopoes.connector.octopoes import OctopoesAPIConnector
 from octopoes.models import ScanLevel, ScanProfileType
 from octopoes.models.exception import ObjectNotFoundException
+from octopoes.models.ooi.findings import RiskLevelSeverity
 from octopoes.models.ooi.reports import HydratedReport
 from octopoes.models.types import get_collapsed_types, type_by_name
 from rocky.bytes_client import get_bytes_client
+from rocky.views.mixins import FindingList
 
 logger = structlog.get_logger(__name__)
 
 
 class DashboardItemView:
-    item: DashboardData | None = None
+    item: DashboardItem | None = None
     data: dict[str, Any] = {}
```

Comment:
This class seems to only hold data as a container. Perhaps create a data class out of it, or a typed dictionary?